demos/fleet_provisioning/fleet_provisioning_with_csr/CMakeLists.txt

```
@@ -13,6 +13,7 @@ include(
 # Demo target.
 add_executable( ${DEMO_NAME}
                 "fleet_provisioning_csr_demo.c"
+                "fleet_provisioning_payload_operations.c"
```

I suggest that using a `_serializer` filename will be easier to understand.
```suggestion
                "fleet_provisioning_payload_operations.c"
```